### PR TITLE
Added possibility to store different emote command with diff image

### PR DESCRIPTION
**Pull request by Leefrost**
Fix for #47

### CODE REVIEW

File: src/commands/add.js [hunk 1]
```diff
@@ -41,17 +41,28 @@ class Command {
...
-  if (emotes.hasOwnProperty(emote)) {
-    return done(null, 'We already have that emote.');
-  }
-
   const image = getImage(message);
   if (!image) {
     return done(null, 'That is not an image.');
   }
 
+  if(emotes[emote] == image){
```
Comment (arya-s) [inline src/commands/add.js]: Use `===` please

File: src/commands/add.js [hunk 2]
```diff
@@ -41,17 +41,28 @@ class Command {
...
+
+  if (emotes.hasOwnProperty(emote)) {
+    console.log("This emoticon is exist in database. Incrementing command name.");
+    let counter = 1;
+    if(emote.match(/(\d+)$/))
+    {
+      counter = parseInt(emote.match(/(\d+)$/)[0], 10);
+      counter +=1;
+  } else {
+      emote = emote + counter;
```
Comment (arya-s) [inline src/commands/add.js]: Use es6 String interpolation here for clearer code: `emote = `${emote}${counter}``

File: src/commands/add.js [hunk 3]
```diff
@@ -60,9 +71,7 @@ const addEmote = function (message, done) {
     }
 
     done(error, 'Added emote.');
```
Comment (arya-s) [inline src/commands/add.js]: Please add a message under what the emote was stored, i.e. ``Added emote as ${emote}``

File: src/commands/add.js [hunk 4]
```diff
@@ -41,17 +41,28 @@ class Command {
...
   if (!image) {
     return done(null, 'That is not an image.');
   }
 
+  if(emotes[emote] == image){
+    return done(null, 'Image with command is already exist in database');
+  }
+
+  if (emotes.hasOwnProperty(emote)) {
+    console.log("This emoticon is exist in database. Incrementing command name.");
```
Comment (arya-s) [inline src/commands/add.js]: We don't need this log here, please remove it.

**Review by arya-s — CHANGES_REQUESTED** [review]
Very nice, thank you for your contribution. I've request minor changes.

**Comment by Leefrost** [comment]
Added fixes after review

**Review by arya-s — APPROVED** [review]

**Comment by arya-s** [comment]
Thank you!

**Pull request merged**